src/test/resources/com/znsio/teswiz/features/jiomeet.feature

```diff
@@ -45,4 +45,11 @@ Feature: In a meeting scenarios
     And "Guest-2" joins the meeting from "jiomeet-chrome" on "web"
     Then "Host" should be able to get to chat window
     When "Host" sends "Hey" chat message
-    Then "Guest-2" should see the chat message on its chat window
\ No newline at end of file
+    Then "Guest-2" should see the chat message on its chat window
+
+#  CONFIG=./configs/jiomeet_local_config.properties TAG="@jiomeet and  @selectNotificationTest" PLATFORM=android ./gradlew run
+    @selectNotificationTest @android
+    Scenario: User can select Notification
```

Comment:
does this seem better?
"user can view and tap on notification bar"
Or
"validating tap on the notification from notification drawer"

Reply:
updated the scenario name

Reply:
is there any change in the scenario name?